Rails/client/package-lock.json

```diff
@@ -0,0 +1,10311 @@
+{
```

Comment:
Do you want to check this in?

Reply:
Yes, the application is the last consumer in the dependency chain so checking in the package lock is safe